pytorch_lightning/core/hooks.py

```diff
@@ -372,6 +372,18 @@ def configure_sharded_model(self) -> None:
 class DataHooks:
     """Hooks to be used for data related stuff."""
 
+    __jit_unused_properties__ = ["prepare_data_per_node"]
+
+    def __init__(self) -> None:
+        """
+        property:
+            prepare_data_per_node:
+                If True, each LOCAL_RANK=0 will call prepare data.
+                Otherwise only NODE_RANK=0, LOCAL_RANK=0 will prepare data.
```

Comment:
```suggestion
        Attributes:
            prepare_data_per_node:
                If True, each LOCAL_RANK=0 will call prepare data.
                Otherwise only NODE_RANK=0, LOCAL_RANK=0 will prepare data.
```
for the Google-style documentation we use, I think this section has to be named "Attributes". 

https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#type-annotations